server/assets.py

```diff
@@ -25,6 +27,9 @@
 
 common_js = Bundle(
     'js/main.js',
+    'js/notebook.js',
+    'lib/notebookjs/notebook.min.js',
```

Comment:
out of this PR scope - but for the future: We should figure out how to load these assets (along with all of the others used for code rendering) only in the code view. It seems unnecessary otherwise. 

I'm thinking assets bundles along the lines of code_js / code_css